src/helpers/extrinsitcs.ts

```diff
@@ -0,0 +1,40 @@
+import { SubmittableExtrinsic } from "@polkadot/api/types"
+import { web3FromAddress } from "@polkadot/extension-dapp"
+
```

Comment:
I think we could also move the contents of `BiddersPage/index.tsx` here, but in another PR.